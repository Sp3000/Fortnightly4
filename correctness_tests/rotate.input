rotate 0 0 1
place 0 0 red
rotate 0 0 1
place 0 0 green
place 1 0 blue
connect 0 0 1 0
rotate 0 0 1
place 1 0 purple
place 0 1 yellow
rotate 0 1 1
place 0 0 purple
place 1 1 yellow
rotate 0 1 2
place -1 1 yellow
connect 0 0 0 1
rotate 0 0 2
place 0 -1 orange
place 1 -1 orange
count
connect 1 0 1 -1
count
rotate 1 0 3
place 0 0 red
place 0 1 red
place 1 0 red
place 1 1 red
halt