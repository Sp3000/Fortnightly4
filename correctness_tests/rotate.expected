Error: rotate 0 0 1
Error: place 0 0 green
Error: place 0 1 yellow
Error: place 1 1 yellow
Error: place -1 1 yellow
Error: place 0 -1 orange
Error: place 1 -1 orange
2
1
Error: place 0 0 red
Error: place 0 1 red
Error: place 1 0 red
Error: place 1 1 red